eslint.config.mjs

```diff
@@ -103,6 +105,23 @@ export default tseslint.config([
     },
 
     settings: {
+      jsdoc: {
+        exemptDestructuredRootsFromChecks: true,
+        contexts: [
+          // We don't want to require documentation of a 'locals' (res.locals) variable
+          // AST Parser: https://github.com/es-joy/jsdoccomment
+          {
+            comment: 'JsdocBlock:not(:has(JsdocTag[tag="param"][name="locals"]))',
```

Comment:
We could consider a similar idea for other nodes that are often destructured. Potentially even a feature request to ignore parameters with certain names?